server/lib/pdf.js

```diff
@@ -9,7 +9,7 @@ import { fetchWithTimeout } from './fetch';
 import logger from './logger';
 
 export const getTransactionPdf = async (transaction, user) => {
-  if (config.pdfService.fetchTransactionsReceipts === false) {
+  if (Boolean(config.pdfService.fetchTransactionsReceipts) === false) {
```

Comment:
That doesn't work, `Boolean('false') is `true`. You have to use our `parseToBoolean` like in similar places.